cli.ts

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env node
+
+import sade from 'sade';
+import { scan } from './scan-task';
+import { usefulDirName } from './utilities';
+import * as path from 'path';
+import { version } from './package.json';
+
+const program = sade('lighthouse-parade');
+
+program.version(version);
+
+program
+  .command(
+    'scan <url> [dataDirectory]',
+    'Crawls the site at the provided URL, recording the lighthouse scores for each URL found. The lighthouse data will be stored in the provided directory, which defaults ./data/YYYY-MM-DDTTZ_HH_MM'
+  )
```

Comment:
What do you think about making this the default command instead of making the user type the subcommand `scan`? https://github.com/lukeed/sade#optsdefault

Reply:
✅